shared/userSchema.ts

```diff
@@ -0,0 +1,47 @@
+import { generateIdFromEntropySize } from 'lucia';
+import { z } from 'zod';
+
+export const userInsertSchema = z.object({
```

Comment:
its possible to derive these from the table definitions https://orm.drizzle.team/docs/zod